WORKSPACE

```diff
@@ -57,21 +57,39 @@ load(
 
 container_repositories()
 
+container_pull(
+    name = "distroless",
```

Comment:
why not use the same image as upstream (debian-base)?

Reply:
In short: size and security patching.
debian-base has a lot more unnecessary stuff which often has CVEs and they often go unpatched for long.
`distroless` is basically scratch with just a handfull of essentials linux added (ca-certs, libc, tzdata, etc) but no shell/package manager/etc.

Reply:
- Security: We still need to patch the other images from upstream too though since we also ship them? If we think distroless is best there we should patch this upstream?
- Size: common base layer should be copy on write anyhow?
¯\\\_(ツ)_/¯

Reply:
we have a longer-term goal of switching away from debian-base: https://github.com/kubernetes/kubernetes/issues/70249

I think `distroless/static` is probably what we want though (ref. https://github.com/kubernetes/kubernetes/issues/70249)?

Reply:
Thanks @ixdy. GitHub isn't letting me mark this as resolved but consider this resolved...